### DISCUSSION TITLE
At what stage of the install.sh of T-Pot does Kibana import the kibana_export.ndjson.zip?

### DISCUSSION THREAD

**Discussion by tadashi-oya** [telekom-security/tpotce#1055] [Q&A] [answered]
At what stage of the install.sh of T-Pot does Kibana import the kibana_export.ndjson.zip?
I read install.sh but could not find it.

I would like to know why the Visualize Library and Dashboard required by T-Pot are available in Kibana immediately after installation.

**Comment by t3chn0m4g3** [comment, marked as answer]
https://github.com/telekom-security/tpotce/blob/e2752458d40feca4d933343134a0c9ee6c1e1590/iso/installer/install.sh#L848

**Reply by tadashi-oya** [reply]
I see that it is directly deployed as an index in Elasticsearch.
Thank you very much. I understand now.
